gusto/time_discretisation.py

```diff
@@ -315,123 +331,55 @@ def apply(self, x_out, x_in):
         x_out.assign(self.x1)
 
 
-class ForwardEuler(ExplicitTimeDiscretisation):
+class ExplicitMultistage(ExplicitTimeDiscretisation):
```

Comment:
I don't mind `ExplicitMultistage`, but have also wondered if `ExplicitRungeKutta` is clearer?

Reply:
I think I may veto this.. I'm keener on multistage (especially in contrast to multistep)